ui/src/app/pages/workspace/workspace-library.tsx

```diff
@@ -131,19 +158,21 @@ export const WorkspaceLibrary = withUserProfile()
   async updateWorkspaces() {
     await this.getAllPublishedWorkspaces();
     await this.getFeaturedWorkspaces();
-    this.filterPublishedWorkspaces();
   }
 
   // Gets all published workspaces, including those configured as 'featured'
   async getAllPublishedWorkspaces() {
+    this.setState({
```

Comment:
React nit (for here and below): setState calls which rely on the current state should be done with the "updater" function parameter. From https://reactjs.org/docs/react-component.html#setstate: "If you need to set the state based on the previous state, read about the updater argument below."

See https://github.com/all-of-us/workbench/blob/f796b9bc2e343bb251fc4373707025741c4e4935/ui/src/app/components/side-nav.tsx#L111 for an example.